lib/src/drm/keysystem/getSupportedKeySystemConfiguration.ts

```diff
@@ -0,0 +1,62 @@
+import type { MediaCapability } from '../common/MediaCapability.js';
+import type { KeySystemConfiguration } from '../common/KeySystemConfiguration.js';
+
+/**
+ * Filters and returns the supported key system configuration for a given system string.
+ *
+ * @param keySystemString - Key system string such as 'com.widevine.alpha'
+ * @param configs - An array of key system configurations.
+ * @returns The first supported configuration or null if none are supported.
+ *
+ * @group DRM
+ * @beta
+ */
+export function getSupportedKeySystemConfiguration(
+	keySystemString: string,
+	configs: KeySystemConfiguration[],
+): { supportedAudio: MediaCapability[] | null; supportedVideo: MediaCapability[] | null } | null {
+	if (!configs || configs.length === 0) {
+		return null;
+	}
+
+	// legacy isTypeSupported from MediaKeys
+	const isTypeSupported = (window.MediaKeys as any)?.isTypeSupported;
```

Comment:
Have this fallback to a function that just returns `false`. That way you don't need to constantly check `typeof isTypeSupported === 'function'`:
```suggestion
	const isTypeSupported = (window.MediaKeys as any)?.isTypeSupported || (...args: any[]) => false;
```